python/pyspark/sql/tests/connect/test_connect_column.py

```diff
@@ -437,7 +436,6 @@ def test_literal_with_unsupported_type(self):
             (0.1, DecimalType()),
             (datetime.date(2022, 12, 13), TimestampType()),
             (datetime.timedelta(1, 2, 3), DateType()),
-            ([1, 2, 3], ArrayType(IntegerType())),
```

Comment:
Let's add test for Array<Array<T>> type as well.

Reply:
arrayType was not support before, here just check the raised `AssertionError `